Kudu.Core/K8SE/K8SEDeploymentHelper.cs

```diff
@@ -139,13 +140,17 @@ public static string GetAppName(HttpContext context)
 
         public static string GetAppType(HttpContext context)
         {
+#if DEBUG
+            foreach (var header in context.Request.Headers)
+            {
+                Console.WriteLine($"**** Debug BuildServer: {header.Key}: {header.Value.ToString()}");
+            }
```

Comment:
I understand the intent here. Instead of using `#if Debug` - we should move to `ILogger` and rely on [IsDevelopmentFlag](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.hosting.hostingenvironmentextensions.isdevelopment?view=aspnetcore-5.0)

Please delete this for now and address it as part of https://github.com/Azure-App-Service/KuduLite/issues/202

Reply:
That's make sense. I love the idea. 
Done.